src/utils/formatters.ts

```diff
@@ -2,29 +2,27 @@ import { BigNumberish, type BigNumber } from 'ethers'
 import { formatUnits, parseUnits } from 'ethers/lib/utils'
 import { formatAmount } from './formatNumber'
 
-export const toDecimals = (value: BigNumberish, decimals?: number | string): number => {
-  return Number(formatUnits(value, decimals))
-}
-
-export const formatDecimals = (value: BigNumberish, decimals?: number | string): string => {
-  return formatAmount(toDecimals(value, decimals))
-}
-
-export const toWei = (value: string, decimals?: number | string): BigNumber => {
-  return parseUnits(value, decimals)
-}
-
 const GWEI = 'gwei'
 
+// safeFormatUnits => "0.000000000001"
```

Comment:
I would suggest to write a proper comment here which explains the difference between `formatVisualAmount` and `safeFormatUnits`.

i.e.
```ts
/**
 *  Formats a GWEI value to decimal with high precision. 
 *  The resulting string is not formatted with respect to the standard amount formatting. Use {@link formatVisualAmount} to format values with amount formatting.
 * @param value in GWEI
 * @param decimals decimals of the value in GWEI, 18 decimals are used if undefined
 * @returns formatted decimal value with high precision
 */
```